lib/bundler/rubygems_ext.rb

```diff
@@ -23,6 +23,21 @@
 module Gem
   @loaded_stacks = Hash.new {|h, k| h[k] = [] }
 
+  class StubSpecification
```

Comment:
Stub specification isn't always defined, and I'd like us to avoid adding even more re-opening of classes we don't own